src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -131,7 +131,7 @@ private void OnAnalysisRequested(object sender, AnalysisRequestEventArgs args)
                 reanalysisProgressHandler = new StatusBarReanalysisProgressHandler(vsStatusBar, logger);
 
                 var message = string.Format(CultureInfo.CurrentCulture, Strings.JobRunner_JobDescription_ReaanalyzeDocs, operations.Length);
-                reanalysisJob = CancellableJobRunner.Start(message, operations,
+                reanalysisJob = CancellableJobRunner.Instance.Start(message, operations,
```

Comment:
This is the only product code that uses the cancellable job runner. I didn't change this class so the runner could be injected as part of this PR, but now that the new abstractions exist it would be possible to do so.